### PR TITLE
Cookbookbat consumable spading data

### PR DESCRIPTION
**Pull request by Malibu-Stacey**
- added adv gain for the 3 food that were missing it.
- added stat gains for all 14 foods and 1 booze and removed the Unspaded tags.
- updated the tier 4 adv gain ranges.

### CODE REVIEW

File: src/data/fullness.txt [hunk 1]
```diff
@@ -91,7 +91,7 @@ blunt cat claw	1	4	awesome	3-5	45-75	0	0	30 Feline Ferocity (+100% weapon dmg)
 bone broth	1	1	awesome	4-5	40-50	0	0	1-2 loose teeth, 3-4 skeleton bones
 bone meal	2	6	awesome	7-9	12-20	12-20	12-20
 boring spaghetti	4	5	good	10-14	0	20-25	0
-Boris's bread	1	1	EPIC	6-8	0	0	0	Unspaded, 30 Inspired Chef (+10 Muscle, +100% Muscle, +100% Meat Drop)
+Boris's bread	1	1	EPIC	6-8	40-80	0	0, 30 Inspired Chef (+10 Muscle, +100% Muscle, +100% Meat Drop)
```
Comment (Veracity0) [inline src/data/fullness.txt:94]: ditto

File: src/data/fullness.txt [hunk 2]
```diff
@@ -469,7 +469,7 @@ hippy herbal tea	1	1	decent	2	0	4-7	0	5 Sleepy (-30% mus)
 holiday cheese log	3	1	good	8-10	10-20	10-20	10-20	Stands Alone (+10 Stench dmg)
 honey-dew	2	2	good	4-7	13-15	0	0	10 Mmmmmelon (+30% item drop)
 honey-drenched ham slice	1	1	decent	1-2	1-5	1-5	1-5
-honey bun of Boris	1	1	EPIC	5-7	0	0	0	Unspaded, 30 Motherly Loved (+5 Muscle, +50% Muscle, +5 Cold Res)
+honey bun of Boris	1	1	EPIC	5-7	20-40	0	0, 30 Motherly Loved (+5 Muscle, +50% Muscle, +5 Cold Res)
```
Comment (Veracity0) [inline src/data/fullness.txt:472]: Need I go on? Fix all of them.

**Review by Veracity0 — CHANGES_REQUESTED** [review]

**Review by midgleyc — APPROVED** [review]
Fixed them myself, because I would also like this change!

Thanks!

**Review by Veracity0 — APPROVED** [review]
All good now.

I'm done with consumption today,but this will be very useful from tomorrow on.

Thanks!

**Pull request merged**